File: src/main/java/net/sf/rails/game/model/StockMarketModel.java
package net.sf.rails.game.model;

import net.sf.rails.game.financial.StockMarket;

public class StockMarketModel extends RailsModel {

    public static final String ID = "StockMarketModel";
    
    private StockMarketModel(StockMarket parent, String id) {
        super(parent, id);
    }

    public static StockMarketModel create (StockMarket parent) {
        return new StockMarketModel(parent, ID);
    }
    
}

File: dotnet/RailsLib.Net/Net/Game/Model/StockMarketModel.cs
using GameLib.Net.Game.Financial;
using System;


namespace GameLib.Net.Game.Model
{
    public class StockMarketModel : RailsModel
    {
        public static readonly string ID = "StockMarketModel";

        private StockMarketModel(StockMarket parent, String id) : base(parent, id)
        {

        }

        public static StockMarketModel Create(StockMarket parent)
        {
            return new StockMarketModel(parent, ID);
        }
    }
}
